Description: Mark tests as TODO which fail on 32bit architectures.
Origin: vendor
Bug: https://github.com/stevieb9/ipc-shareable/issues/14
Bug-Debian: https://bugs.debian.org/997829
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2022-01-22

--- a/t/65-seg_size.t
+++ b/t/65-seg_size.t
@@ -39,6 +39,9 @@
 }
 
 # beyond RAM limits
+TODO: {
+    local $TODO = 'Fails on 32bit systems, cf. #997829';
+
 {
     my $size_ok = eval {
         tie my $var, 'IPC::Shareable', {
@@ -53,6 +56,7 @@
 
     like $@, qr/Cannot allocate memory|Out of memory|Invalid argument/, "...and error is sane";
 }
+} # end TODO
 
 my $k = tie my $sv, 'IPC::Shareable', {
     create => 1,
